B827EBFFFE93380F.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFE93380F", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-eu", 
```

Comment:
This is not the correct address. If you want to connect to the European router, then it should be `router.eu.thethings.network`. The full list is here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses

Reply:
Thank you very much!